### ISSUE TITLE
UnknownPropertyException on frontend with Craft 3.7.28 and Formie 1.5.4

### ISSUE THREAD

**Issue by RyanRoberts** [verbb/formie#702]
**Description**

Getting the [following error](https://media.yep.digital/NDHU8i) on the frontend as per the title:

```
yii\base\UnknownPropertyException: Getting unknown property: craft\models\FieldLayoutTab::settings in /Users/ryan/Git/client/vendor/yiisoft/yii2/base/Component.php:154
Stack trace:
#0 /Users/ryan/Git/client/vendor/verbb/formie/src/elements/Form.php(645): yii\base\Component->__get('settings')
#1 /Users/ryan/Git/client/vendor/verbb/formie/src/elements/Form.php(676): verbb\formie\elements\Form->hasButtonConditions()
#2 /Users/ryan/Git/client/vendor/verbb/formie/src/elements/Form.php(1297): verbb\formie\elements\Form->hasConditions()
#3 /Users/ryan/Git/client/vendor/verbb/formie/src/services/Rendering.php(90): verbb\formie\elements\Form->getFrontEndJsVariables()
#4 /Users/ryan/Git/client/vendor/verbb/formie/src/variables/Formie.php(103): verbb\formie\services\Rendering->renderForm(Object(verbb\formie\elements\Form), NULL)
#5 /Users/ryan/Git/client/vendor/twig/twig/src/Extension/CoreExtension.php(1564): verbb\formie\variables\Formie->renderForm(Object(verbb\formie\elements\Form))
#6 /Users/ryan/Git/client/vendor/craftcms/cms/src/helpers/Template.php(106): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderForm', Array, 'method', false, false)
#7 /Users/ryan/Git/client/storage/runtime/compiled_templates/56/56324d9d0e2d4a3e0dd3132ed3ac894c7d2205b61050c66ca9dac3d26ffdccdd.php(131): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderForm', Array, 'method')
#8 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(405): __TwigTemplate_c77bea192a4055fa569165e15b03babcda63f8156eb1ae02988462f375d8327b->doDisplay(Array, Array)
#9 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#10 /Users/ryan/Git/client/storage/runtime/compiled_templates/79/791aa503a7bc1da6e307d020ff3be7547ce25bbb08ece8e83aba4d0efbbe1561.php(80): Twig\Template->display(Array)
#11 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(182): __TwigTemplate_3d0a4c39152d34e2238bcfde0f385210cdcbbffe5f6e610ea397a76151cb7a3d->block_main(Array, Array)
#12 /Users/ryan/Git/client/storage/runtime/compiled_templates/08/089ec019625920112ef9e4ef4fe2c067d616ae96a0a8ea1f2fafdfe5e2e98dcf.php(61): Twig\Template->displayBlock('main', Array, Array)
#13 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(182): __TwigTemplate_6b4a59d8f14adb1a5ffd656d822e63bb0824b84fc36d02a18f090f262384751c->block_body(Array, Array)
#14 /Users/ryan/Git/client/storage/runtime/compiled_templates/19/190ffda65a6a814f5b78a7fa9592b0487e0743976a0df6ee7a5bff66db9e44f7.php(171): Twig\Template->displayBlock('body', Array, Array)
#15 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(182): __TwigTemplate_aaf533b8ebe43322a71ef6fa492d23a7ef792cceafb7b6b2744c07ab1ccf0c6b->block_htmlPage(Array, Array)
#16 /Users/ryan/Git/client/storage/runtime/compiled_templates/a4/a4ae799db1c83a63e1c7aaf185e9bad7fca510be6836e37181bb3ce2ae43fad4.php(113): Twig\Template->displayBlock('htmlPage', Array, Array)
#17 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(405): __TwigTemplate_435c22cba08e30447dbddf2e938acc800d2c116b248dff93e37e5998a4c6c641->doDisplay(Array, Array)
#18 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#19 /Users/ryan/Git/client/storage/runtime/compiled_templates/19/190ffda65a6a814f5b78a7fa9592b0487e0743976a0df6ee7a5bff66db9e44f7.php(51): Twig\Template->display(Array, Array)
#20 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(405): __TwigTemplate_aaf533b8ebe43322a71ef6fa492d23a7ef792cceafb7b6b2744c07ab1ccf0c6b->doDisplay(Array, Array)
#21 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#22 /Users/ryan/Git/client/storage/runtime/compiled_templates/08/089ec019625920112ef9e4ef4fe2c067d616ae96a0a8ea1f2fafdfe5e2e98dcf.php(44): Twig\Template->display(Array, Array)
#23 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(405): __TwigTemplate_6b4a59d8f14adb1a5ffd656d822e63bb0824b84fc36d02a18f090f262384751c->doDisplay(Array, Array)
#24 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#25 /Users/ryan/Git/client/storage/runtime/compiled_templates/79/791aa503a7bc1da6e307d020ff3be7547ce25bbb08ece8e83aba4d0efbbe1561.php(61): Twig\Template->display(Array, Array)
#26 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(405): __TwigTemplate_3d0a4c39152d34e2238bcfde0f385210cdcbbffe5f6e610ea397a76151cb7a3d->doDisplay(Array, Array)
#27 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling(Array, Array)
#28 /Users/ryan/Git/client/vendor/twig/twig/src/Template.php(390): Twig\Template->display(Array)
#29 /Users/ryan/Git/client/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render(Array, Array)
#30 /Users/ryan/Git/client/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render(Array)
#31 /Users/ryan/Git/client/vendor/craftcms/cms/src/web/View.php(408): Twig\Environment->render('_routers/page', Array)
#32 /Users/ryan/Git/client/vendor/craftcms/cms/src/web/View.php(461): craft\web\View->renderTemplate('_routers/page', Array)
#33 /Users/ryan/Git/client/vendor/craftcms/cms/src/web/Controller.php(201): craft\web\View->renderPageTemplate('_routers/page', Array, 'site')
#34 /Users/ryan/Git/client/vendor/craftcms/cms/src/controllers/TemplatesController.php(102): craft\web\Controller->renderTemplate('_routers/page', Array)
#35 [internal function]: craft\controllers\TemplatesController->actionRender('_routers/page', Array)
#36 /Users/ryan/Git/client/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#37 /Users/ryan/Git/client/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#38 /Users/ryan/Git/client/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('render', Array)
#39 /Users/ryan/Git/client/vendor/craftcms/cms/src/web/Application.php(287): yii\base\Module->runAction('templates/rende...', Array)
#40 /Users/ryan/Git/client/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('templates/rende...', Array)
#41 /Users/ryan/Git/client/vendor/craftcms/cms/src/web/Application.php(272): yii\web\Application->handleRequest(Object(craft\web\Request))
#42 /Users/ryan/Git/client/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#43 /Users/ryan/Git/client/public/index.php(22): yii\base\Application->run()
#44 /Users/ryan/.composer/vendor/laravel/valet/server.php(234): require('/Users/ryan/Git...')
#45 {main}```


**Steps to reproduce**
1. Load frontend with form present

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.4
- Craft version: 3.7.28
- Multi-site: No

**Additional context**

Currently this is running locally in Valet

**Comment by nickdunn** [comment]
~~I'm seeing something very similar after upgrading to 1.5.3 so I'm assuming it's the same thing:~~

```
php_1     | 2022-01-06 13:27:39 [172.22.0.1][7516][541f1e66d81fefc4922fb4d951ac939b][error][TypeError] TypeError: craft\models\FieldLayout::getTabs(): Return value must be of type array, null returned in /var/www/craft/vendor/craftcms/cms/src/models/FieldLayout.php:258
php_1     | Stack trace:
php_1     | #0 /var/www/craft/vendor/verbb/formie/src/models/FieldLayout.php(87): craft\models\FieldLayout->getTabs()
php_1     | #1 /var/www/craft/vendor/verbb/formie/src/elements/Form.php(587): verbb\formie\models\FieldLayout->getTabs()
php_1     | #2 /var/www/craft/vendor/verbb/formie/src/elements/Form.php(686): verbb\formie\elements\Form->getPages()
php_1     | #3 /var/www/craft/vendor/verbb/formie/src/elements/Form.php(1260): verbb\formie\elements\Form->hasMultiplePages()
php_1     | #4 /var/www/craft/vendor/verbb/formie/src/services/Rendering.php(90): verbb\formie\elements\Form->getFrontEndJsVariables()
php_1     | #5 /var/www/craft/vendor/verbb/formie/src/variables/Formie.php(103): verbb\formie\services\Rendering->renderForm(Object(verbb\formie\elements\Form), NULL)
php_1     | #6 /var/www/craft/vendor/twig/twig/src/Extension/CoreExtension.php(1564): verbb\formie\variables\Formie->renderForm(Object(verbb\formie\elements\Form))
php_1     | #7 /var/www/craft/vendor/craftcms/cms/src/helpers/Template.php(106): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderForm', Array, 'method', false, false)
php_1     | #8 /var/www/craft/storage/runtime/compiled_templates/ed/edb31fdacb4c32f6203d6a95c07be0f67bd49721df9f454e2a323338f1635085.php(42): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderForm', Array,
```

- Plugin version: 1.5.3-
-  Craft version: 3.7.28

**Comment by nickdunn** [comment]
Updated to 1.5.4 and now getting same exception as @RyanRoberts.

**Comment by DavidKabelitz** [comment]
The same here

**Comment by janeschindler** [comment]
me too

**Comment by JohnCBent** [comment]
I'm also seeing this issue.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.5](https://github.com/verbb/formie/releases/tag/1.5.5)
